apps/etterlatte-vilkaarsvurdering/.nais/dev.yaml

```diff
@@ -69,34 +69,15 @@ spec:
         thresholdPercentage: 90
     max: 4
     min: 2
-  env:
-    - name: ETTERLATTE_BEHANDLING_URL
-      value: http://etterlatte-behandling
-    - name: ETTERLATTE_BEHANDLING_CLIENT_ID
-      value: 59967ac8-009c-492e-a618-e5a0f6b3e4e4
-    - name: ETTERLATTE_GRUNNLAG_URL
-      value: http://etterlatte-grunnlag/api
-    - name: ETTERLATTE_GRUNNLAG_CLIENT_ID
-      value: ce96a301-13db-4409-b277-5b27f464d08b
   envFrom:
     - secret: my-application-unleash-api-token
   accessPolicy:
     inbound:
-      rules:
-        - application: etterlatte-saksbehandling-ui-lokal # for å åpne for lokal utv. Ikke kopier denne til prod.yaml
-        - application: etterlatte-saksbehandling-ui
-        - application: etterlatte-beregning
+      rules: #TODO: sjekk alle her ligger i behandling
         - application: etterlatte-behandling
-        - application: etterlatte-brev-api
-        - application: etterlatte-vedtaksvurdering
+        - application: etterlatte-saksbehandling-ui-lokal # for å åpne for lokal utv. Ikke kopier denne til prod.yaml
         - application: etterlatte-vilkaarsvurdering-kafka
```

Comment:
Inn her for vilkaarsvurdering-kafka kan vi vel også slette?

Reply:
jepp! https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/5618/commits/dc79ace18b2c74a935a3e7713a39dc6bbf6281be